.github/workflows/pull-requests.yml

```
@@ -112,7 +113,8 @@ jobs:
             [ ${{ needs.e2e-tests-plugins.result }} != 'success' ] ||
             [ ${{ needs.api-library-tests.result }} != 'success' ] ||
             [ ${{ needs.toolbox-tests.result }} != 'success' ] ||
-            [ ${{ needs.integration-tests-on-prem.result }} != 'success' ]; then
+            [ ${{ needs.federation-tests.result }} != 'success' ]; ||
```

Or maybe not, you have a typo! 😆 

```suggestion
            [ ${{ needs.federation-tests.result }} != 'success' ] ||
```